src/workloads/execution/DensifyTimeseriesCollection.yml

```diff
@@ -83,13 +83,13 @@ Actors:
     Collection: *coll
     Threads: 1
     CollectionCount: 1
-    DocumentCount: 10000
+    DocumentCount: 100000
     BatchSize: *batchSize
     Document:
       partitionKey: {^RandomInt: {min: 1, max: 100}}
-      timestamp: {^RandomDate: {min: "2021-01-01", max: "2023-01-01"}}
+      timestamp: {^RandomDate: {min: "2021-01-01T00:00:00.000", max: "2021-01-01T00:01:00.000"}}
```

Comment:
You may want to run a perf test just to double check this recovers the lost performance & produces reasonable bucketing.  Going backwards in time relative to the first observed time still causes a bucket closure.

Reply:
Sounds good I think I'll change it to ^IncDate

Reply:
Also checked the bucketing for a few different runs, it seems reasonable and definitely much better than before